main.go

```diff
@@ -200,7 +199,7 @@ func router(hh httpHandlers) http.Handler {
 
 	//TODO check writers /__health endpoint?
```

Comment:
This is a cool way to return gtg.Status but cant get the code to execute this part. My goodToGo just returns a blank page atm and if If I add this to the goodToGo func the compiler complains that I have changed the method signature